operator/api/v1beta1/helmrelease_types.go

```diff
@@ -31,7 +31,6 @@ type HelmReleaseStatusReason string
 
 const (
 	ReasonDeleteError     HelmReleaseStatusReason = "DeleteError"
```

Comment:
The DeleteError const can be removed as well.

Reply:
Actually, maybe we want to keep that one and set a failure condition here: https://github.com/Wattpad/ship-it/pull/90/files#diff-a76fcebd9b2cb18f81fa0b2611d79357R190

Reply:
I'll leave as is for now.